crates/ruff_linter/src/rules/pyupgrade/rules/pep695/use_pep695_type_alias.rs

```diff
@@ -27,13 +26,13 @@ use crate::settings::types::PythonVersion;
 ///
 /// ## Known problems
 /// [PEP 695] uses inferred variance for type parameters, instead of the
-/// `covariant` and `contravariant` keywords used by `TypeParam` variables. As
-/// such, rewriting a `TypeParam` variable to a `type` alias may change its
+/// `covariant` and `contravariant` keywords used by `TypeVar` variables. As
+/// such, rewriting a `TypeVar` variable to a `type` alias may change its
 /// variance.
```

Comment:
This issue predates your PR, but the wording here doesn't really make sense to me; I'd change it to

```suggestion
/// `covariant` and `contravariant` keywords used by `TypeVar` variables. As
/// such, rewriting a type alias using a PEP-695 `type` statement may change
/// the variance of the alias's type parameters.
```